src/components/NftPreview/NftPreviewLabel.tsx

```diff
@@ -2,49 +2,119 @@ import colors from 'components/core/colors';
 import styled from 'styled-components';
```

Comment:
This file underwent changes since we show a different label based on whether or not we're showing a POAP.

I also refactored this to use Relay patterns